src/shared/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -115,37 +158,201 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
+      player.muted(false)
+      debouncedVolumeChange.current(volume)
+    } else {
+      player.muted(true)
     }
-  }, [player, playerVolume])
+  }, [isInBackground, player, volume])
+
+  const handleChangeVolume = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setVolume(Number(e.target.value))
+  }
+
+  const handlePlayPause = () => (isPlaying ? player?.pause() : player?.play())
+
+  const handleMute = () => {
+    if (volume === 0) {
+      setVolume(cachedPlayerVolume || 0.05)
+    } else {
+      setVolume(0)
+    }
+  }
+
+  const handlePictureInPicture = () => {
+    if (document.pictureInPictureElement) {
+      // @ts-ignore @types/video.js is outdated and doesn't provide types for some newer video.js features
+      player.exitPictureInPicture()
+    } else {
+      if (document.pictureInPictureEnabled) {
+        // @ts-ignore @types/video.js is outdated and doesn't provide types for some newer video.js features
+        player.requestPictureInPicture()
+      }
+    }
+  }
+
+  const handleFullScreen = () => {
+    if (player?.isFullscreen()) {
+      player?.exitFullscreen()
+    } else {
+      player?.requestFullscreen()
+    }
+  }
+
+  const renderVolumeButton = () => {
+    if (volume === 0) {
+      return <StyledSvgPlayerSoundOff />
+    } else {
+      return volume <= 0.5 ? <SvgPlayerSoundHalf /> : <SvgPlayerSoundOn />
+    }
+  }
 
   return (
-    <Container className={className} isInBackground={isInBackground}>
+    <Container isFullScreen={isFullScreen} className={className} isInBackground={isInBackground}>
       {displayPlayOverlay && (
         <PlayOverlay onClick={handlePlayOverlayClick}>
           <SvgOutlineVideo width={72} height={72} viewBox="0 0 24 24" />
         </PlayOverlay>
       )}
       <div data-vjs-player>
         <video ref={playerRef} className="video-js" />
+        {!isInBackground && !playOverlayVisible && (
+          <CustomControls isFullScreen={isFullScreen}>
+            <ControlButton onClick={handlePlayPause}>
+              {isPlaying ? <SvgPlayerPause /> : <SvgPlayerPlay />}
+            </ControlButton>
+            <VolumeControl>
+              <VolumeButton onClick={handleMute}>{renderVolumeButton()}</VolumeButton>
+              <VolumeSliderContainer>
+                <VolumeSlider
+                  step={VOLUME_STEP}
+                  max={1}
+                  min={0}
+                  value={volume}
+                  onChange={handleChangeVolume}
+                  type="range"
+                />
+              </VolumeSliderContainer>
+            </VolumeControl>
+            <CurrentTime variant="body2">
+              {formatDurationShort(videoTime)} / {formatDurationShort(Math.floor(player?.duration() || 0))}
+            </CurrentTime>
+            <ScreenControls>
+              {isPictureInPictureSupported && (
+                <ControlButton onClick={handlePictureInPicture}>
+                  {isPiPEnabled ? <SvgPlayerPipDisable /> : <SvgPlayerPip />}
```

Comment:
we should name everything picture in picture as PiP see in this line there is `SvgPlayerPip` and `SvgPlayerPipDisable`